Sources/SwiftDocC/Infrastructure/Workspace/DefaultAvailability.swift

```diff
@@ -73,7 +131,9 @@ public struct DefaultAvailability: Codable, Equatable {
     /// Creates a default availability module.
     /// - Parameter modules: A map of modules and the default platform availability for symbols in that module.
     public init(with modules: [String: [ModuleAvailability]]) {
-        self.modules = modules
+        self.modules = modules.mapValues { platformAvailabilities -> [DefaultAvailability.ModuleAvailability] in
+            platformAvailabilities
+        }
```

Comment:
Why is this change necessary? The closure passed to `mapValues` returns the unmodified value.

Reply:
I updated this to be:
```swift
 public init(with modules: [String: [ModuleAvailability]]) {
            self.modules = modules.mapValues { platformAvailabilities -> [DefaultAvailability.ModuleAvailability] in
            // If a module doesn't contain default availability information for any of the fallback platforms,
            // infer it from the corresponding mapped value.
            platformAvailabilities + DefaultAvailability.fallbackPlatforms.compactMap { (platform, fallbackPlatform) in
                if !platformAvailabilities.contains(where: { $0.platformName == platform }),
                   let fallbackAvailability = platformAvailabilities.first(where: { $0.platformName == fallbackPlatform }),
                   let fallbackIntroducedVersion = fallbackAvailability.introducedVersion
                {
                    return DefaultAvailability.ModuleAvailability(
                        platformName: platform,
                        platformVersion: fallbackIntroducedVersion
                    )
                }
                return nil
            }
        }
    }
```